deeplenstronomy/utils.py

```diff
@@ -205,7 +207,15 @@ def draw_from_user_dist(filename, size, mode, step=10, df=None):
         
     else:
         raise NotImplementedError("unexpected mode passed, must be 'sample' or 'interpolate'")
-            
+    if type(params) == str:
```

Comment:
instead of `if type(params) == str:`, it's better to do `if isinstance(params, str):`

Reply:
resolved by 19d8f5b